src/main/java/com/spaceuptech/space_api/sql/SQL.java

```diff
@@ -0,0 +1,71 @@
+package com.spaceuptech.space_api.sql;
+import com.spaceuptech.space_api.proto.Meta;
+import com.spaceuptech.space_api.utils.Config;
+import com.spaceuptech.space_api.utils.Transport;
+import com.spaceuptech.space_api.utils.Utils;
+
+public class SQL {
+    private Config config;
+    private String db;
+
+    public SQL(String db, Config config) {
+        this.db = db;
+        this.config = config;
+    }
+
+    // CREATE
+    public Insert insert(String collection) {
+        return new Insert(this.db, this.config, collection);
+    }
+
+
+    // READ
+    public Get get(String collection) {
+        return new Get(this.db, this.config, collection, "all");
+    }
+
+    public Get getOne(String collection) {
+        return new Get(this.db, this.config, collection, "one");
+    }
+
+
+    // UPDATE
+    public Update update(String collection) {
+        return new Update(this.db, this.config, collection, "all");
+    }
+
+    public Update upsert(String collection) {
+        return new Update(this.db, this.config, collection, "upsert");
+    }
+
+
+    // DELETE
+    public Delete delete(String collection) {
+        return new Delete(this.db, this.config, collection, "all");
+    }
+
+    public void profile(String id, Utils.ResponseListener listener) {
+        Meta meta = Transport.makeMeta(config.projectId, null, db, config.token);
+        Transport.profile(config.stub, id, meta, listener);
+    }
+
+    public void profiles(Utils.ResponseListener listener) {
+        Meta meta = Transport.makeMeta(config.projectId, null, db, config.token);
+        Transport.profiles(config.stub, meta, listener);
+    }
+
+    public void editProfile(String id, String email, String name, String password, Utils.ResponseListener listener) {
```

Comment:
`name`, `email` and `password` should be taken as an object.

Reply:
Or maybe we should make a builder instead

Reply:
Added a Class